### DISCUSSION TITLE
AIR APK debug not working with ADT / Flashdevelop

### DISCUSSION THREAD

**Discussion by yvant** [airsdk/Adobe-Runtime-Support#2520] [Q&A] [answered]
Hello, 

I'm using FlashDevelop to buld my AIR projects for mobile, but I can't run the debug build on Android anymore. When I push the run button, the cmd prompt opens up, compiles and streams the application. Here is what I get on command prompt:

> Installing dist\domination-debug.apk on the device...
> Performing Streamed Install 
> Success
> Strating application on the device for debugging...
> Starting: Intent {cmp=air.net.yvant.dominatio,/.AppEntry}
> Errot type 3
> Error: Activity class {air.net.yvant.domination/air.net.yvant.domination.AppEntry} does not exist.

Is there a way to fix this?

**Comment by marchbold** [comment, marked as answer]
Main AIR activity name has changed more recently, your scripts in FlashDevelop may launch them directly? Try searching for `AppEntry` and replacing it in your scripts with `AIRAppEntry`.

**Comment by yvant** [comment]
Thanks, I changed it on run.bat and it's working like a charm now!
